Guides/NUI/Components: Added namespace info and deprected note (#1274)
Samsung/tizen-docs: docs/application/dotnet/guides/nui/nui-components/Switch.md

![SwitchProperty](./media/SwitchProperty.PNG)

> ## Add namespace using
> Switch is defined in namespace Tizen.NUI.Components.

Reviewer: Please verify the following suggestion for technical accuracy.

```suggestion
## Add namespace
To implement switch, add namespace `using Tizen.NUI.Components` in your application:
```
